glasses618/rails101
# [Feature] Initialize a rails project with bootstrap

- Feature:

1. Add bootstrap to the project
2. Add jQuery to the project
3. Add welcome controller
4. Add navbar and footer 
5. Add flash function 


- How to test:
Run rails server `$rails s`
Open http://localhost:3000 with browser. 
In root page, the navbar and footer will appear.
The green flash message "早安！你好！ " will show.
